src/services/donations/donations.hooks.js

```diff
@@ -418,6 +419,34 @@ const setLessThanCutoffHook = () => async context => {
   return context;
 };
 
+const addProjectToDac = () => async context => {
+  if (
+    !context.result.delegateTypeId ||
+    !context.result.intendedProjectType ||
+    !context.result.intendedProjectTypeId
+  ) {
+    // Just continue if it's a delegation otherwise return
+    return context;
+  }
+  const dacId = context.result.delegateTypeId;
+  const projectObjectId = context.result.intendedProjectTypeId;
+  const dacsService = context.app.service('dacs');
+  const dacModel = dacsService.Model;
+  let campaignId;
+  switch (context.result.intendedProjectType) {
+    case 'campaign':
+      campaignId = projectObjectId;
+      break;
+    case 'milestone':
+      campaignId = (await context.app.service('milestones').get(projectObjectId)).campaignId;
```

Comment:
Use Model and use $select please

Reply:
Any update?

Reply:
@aminlatifi 
Sorry I missed this comment, now it changed